test/execflags/jhh/helper.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3
```

Comment:
Note that I don't think random python scripts need a `.notest`. I think that'd only be needed for `.chpl` files